packages/cfpb-forms/src/organisms/multiselect.less

```diff
@@ -125,6 +129,11 @@ select.o-multiselect {
         }
 
         .a-label {
+            box-sizing: border-box;
+            padding: unit( 5px / @base-font-size-px, em )
```

Comment:
Since this is taking up four lines anyway, let's maybe spell out `padding-top`, `padding-right`, etc. and drop the shorthand so we don't have to remember what order the shorthand goes in.